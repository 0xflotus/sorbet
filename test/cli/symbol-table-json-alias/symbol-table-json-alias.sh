main/sorbet test/cli/symbol-table-json-alias/symbol-table-json-alias.rb -p symbol-table-json | grep "aliasTo"
