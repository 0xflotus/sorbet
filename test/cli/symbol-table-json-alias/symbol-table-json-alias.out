   "aliasTo": 25820
   "aliasTo": 7
       "aliasTo": 16
